student.rb

```diff
@@ -1,9 +1,12 @@
 require './person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom, age, name = 'unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
```

Comment:
- [ ] Please add a method that serves as the 'belongs-to' side (a student belongs to a classroom).
hint: see this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships).
it will be like this:

```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Reply:
I have kindly added the method that serves as the belongs-to side